src/comments.ts

```diff
@@ -81,6 +82,14 @@ and I'll merge this PR almost instantly. Thanks for helping out! :heart:
 ${otherOwners.length === 0 ? "" : `
 (${otherOwners.map(o => "@" + o).join(", ")}: you can do this too.)`}`});
 
+export const WaitUntilMergeIsOK = (user: string, abbrOid: string) => ({
+    // at most one reminder per update
+    tag: `wait-for-merge-offer-${abbrOid}`,
+    status: `Hi @${user}, I can't accept a merge request until the PR has a green CI and was appropriately reviewed. I will let you know once that happens.
```

Comment:
Yeah, I like this wording 👍🏻

Reply:
Yeah, slightly tweaked Ryan's version since there are other reasons besides review that can make it wait for a merge offer...